javascript/javascript1/week4/main.js

```diff
@@ -0,0 +1,97 @@
+// Voice Assistant
+
+
+let personName = "";
+ const todoList = [];
+
+ function getReply(command) {
+    if (command === "" || typeof command !== "string") {
+        alert("Give some command Please!");
+    }
+    else if (command.includes("Hello my name is") && personName == "") {
+        personName = command.split(" ").pop();
+        return `Nice to meet you ${personName}`;
+
+    } else if (command.includes("Hello my name is") && personName != "") {
+        return `${personName} Hi again!`;
+
+    } else if (command === "what is my name?") {
+        return `Your name is ${personName}`;
+
+     } if (command === "Add fishing to my todo") {
+         todoList.push("fishing");
+         return todoList[0] + "added to your todoList.";
```

Comment:
It would be great if you can write a function which would check which todo is written in the command and it would add that particular todo to `todoList`. You would call the function in all cases where the command is `Add ____ to my todo`.
As an example: In that case `getReply("Add baking to my todo")` would also work